.env.example

```diff
@@ -106,3 +106,7 @@ REACT_APP_PROFILE_TUTORIAL_LINK=www.blockpower.link/platform_editprofile
 
 # Minimum number of characters in combined tripler first and last name for search
 REACT_APP_TRIPLER_SEARCH_MIN_CHARS=3
+
+# Title and content for the /locked page
+REACT_APP_LOCKED_PAGE_TITLE="Your account needs further review"
+REACT_APP_LOCKED_PAGE_HTML="<p>Reminder: Your Vote Tripler's information must match the age, city, and full name of the Vote Tripler on the list.</p> <p>Reminder: The 3 voters for each Vote Tripler must be 3 unique names for family and friends fo the Vote Tripler.  Your Vote Triplers may receive a call to verify voters.</p> <p>Please watch these videos carefully: <a href=\"https://bpwr.link/videos\">https://bpwr.link/videos</a></p>"
```

Comment:
looks like there's a typo in the 2nd line: "family and friends **fo** the Vote Tripler"

Reply:
also, could you document these in the README as well?